java/form/src/org/netbeans/modules/form/JavaCodeGenerator.java

```diff
@@ -130,6 +131,7 @@ class JavaCodeGenerator extends CodeGenerator {
     static final int ANONYMOUS_INNERCLASSES = 0;
     static final int CEDL_INNERCLASS = 1;
     static final int CEDL_MAINCLASS = 2;
```

Comment:
Could someone tell me what "CEDL" stands for?

Reply:
i was wondering the same until I found this comment:
https://github.com/apache/netbeans/blob/68be4fcc5d1cef14e87d5c421de1bafd18025ee9/java/form/src/org/netbeans/modules/form/Event.java#L98

Reply:
OK, thanks!